src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/Deflater.cs

```diff
@@ -134,9 +134,12 @@ internal int GetDeflateOutput(byte[] outputBuffer)
 
         private unsafe ZErrorCode ReadDeflateOutput(byte[] outputBuffer, ZFlushCode flushCode, out int bytesRead)
         {
+            Debug.Assert(null != outputBuffer, "Can't pass in a null output buffer!");
```

Comment:
> So what prevents you to fix the assert? ;-)

I believe he's trying to remain consistent with the rest of them, and avoid changing those others in this PR as they're not related to its primary focus.